src/components/tasks/card/index.tsx

```diff
@@ -24,23 +17,31 @@ const Card: FC<Props> = ({ content }) => {
 
   const ownerProfilePic = `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}${ownerId}/img.png`;
 
+  const taskData = {
+    Estimated_completion: endsOn,
+    Started: startedOn,
+    Status: status,
+    Owner: ownerId,
+  };
+  const taskArray: any[] = [];
+  function getTask() {
+    // eslint-disable-next-line array-callback-return
```

Comment:
not needed, please remove it.

Reply:
Converted it to arrow function and used forEach instead of map to get rid of this comment.

https://github.com/Real-Dev-Squad/website-status/commit/f84f9afa7df2679012784eb4f1b1558a2496fdb4